common/Scripts/new-package.sh

```diff
@@ -20,8 +20,8 @@ fi
 YAUTO=$(git rev-parse --show-toplevel)/common/Scripts/yauto.py
 
 # Basic repo name linting check
-if [[ ! "${PACKAGE}" =~ ^[a-z0-9]+(-[a-z0-9]+)*$ ]]; then
-    echo "Package names are restricted to US ASCII lowercase letters, numbers and dashes."
+if [[ ! "${PACKAGE}" =~ ^[a-z0-9.]+(-.[a-z0-9.]+)*$ ]]; then
```

Comment:
```suggestion
if [[ ! "${PACKAGE}" =~ ^[a-z0-9.]+(-[a-z0-9.]+)*$ ]]; then
```
I don't think we want to match any character here (and require two characters after the hyphen.

Reply:
E.g. this doesn't match `libgtkmm-3`, but DOES match `python-$e`